setup.py

```diff
@@ -56,8 +56,9 @@
             # AT2L0 and its simulator:
             'ioc-lfe-at2l0-calc=solid_attenuator.ioc_lfe_at2l0_calc.__main__:main',  # noqa
             'ioc-sim-at2l0=solid_attenuator.ioc_sim_at2l0.__main__:main',
-            # SXR solid attenuator entrypoint, including AT2K2:
+            # SXR solid attenuator entrypoint, including AT2K2 and AT1K2:
             'ioc-satt-ladder-calc=solid_attenuator.ioc_kfe_at1k4_calc.__main__:main',  # noqa
+            'ioc-kfe-at1k2-calc=solid_attenuator.ioc_kfe_at1k2_calc.__main__:main',
```

Comment:
Stick a `#  noqa` ("no quality assurance" checks like linting) on the end of this and flake8 should be quiet about the line

Reply:
Builds are still failing unfortunately